docs/basic-docs/atomicdex-api-20-dev/get_locked_amount.md

```diff
@@ -0,0 +1,86 @@
+# get\_locked\_amount
+
+The `get_locked_amount` method returns the amount of a coin which is currently locked by a swap which is in progress. If the coin queried is not activated, a `NoSuchCoin` error will be returned.
+
+
+#### Arguments
+
+| Parameter | Type   | Description                                                          |
+| --------- | ------ | -------------------------------------------------------------------- |
+| coin      | string | The ticker of the coin you want to query.                            |
+
+
+#### Response
+
+| Parameter                    | Type            | Description                                                                                                    |
+| ---------------------------- | --------------- | -------------------------------------------------------------------------------------------------------------------- |
+| id                           | integer         | Identifies a response to allow matching it with a request. Defaults to `null` if `id` not provided in request.       |
+| coin                         | string          | The ticker of the coin you queried.                                                                                  |
+| locked_amount                | object          | An object cointaining the locked amount in decimal, fraction and rational formats.                                   |
+| locked_amount.decimal        | numeric string  | The locked amount in [decimal format](https://www.mathsisfun.com/definitions/decimal.html).                          |
+| locked_amount.rational       | rational object | The locked amount in [rational format](../atomicdex-api-legacy/rational_number_note.md).                             |
+| locked_amount.fraction       | fraction object | The locked amount in [fraction format](https://www.mathsisfun.com/definitions/fraction.html).                        |
+| locked_amount.fraction.numer | numeric string  | The [numerator](https://www.mathsisfun.com/definitions/numerator.html) for the locked amount in fraction format.     |
+| locked_amount.fraction.denom | numeric string  | The [denominator](https://www.mathsisfun.com/definitions/denominator.html) for the locked amount in fraction format. |
```

Comment:
I think numer and denom description is excessive here. Numbers format should be described in separate docs section, as it is common for all RPCs.

Reply:
I agree - this will be handled as we upgrade docs to new layouts.
related: https://github.com/KomodoPlatform/developer-docs/issues/426

Reply:
Before removing them I'd like to have a doc to link to for them (and other places where the structure is used) so its a "whole of repo" update which will be done separately. Leaving them in here acts as a placeholder for now until site wide update can be done.

Reply:
I removed them anyway :+1: